docs/administration.md

```diff
@@ -97,3 +97,58 @@ Note: This will only work when the replication factor is >= 2
 - Modify the cluster config file to replace the previous node address with the new node address.
 - Restart the cluster
 - Wait for the 1st sync (10 minutes) to validate Index connections
+
+#### Metrics
+
+Pilosa can be configured to emit metrics on its internal processes in 2 formats, Expvar or StatsD.  Metric recording is disabled by default and enabled through the configuration file.  
+Through the configuration file choose the metric type, host to receive events, and polling interval for runtime metrics.
```

Comment:
`Through the configuration file` is confusing to me because configuration options can be supplied vie command line arguments or environment variables. Are metrics configurations restricted to the config file?